### PR TITLE
Update Firefox data for javascript.statements.import.service_worker_support

### LINKED ISSUE #23160: javascript.statements.import - "Available in service workers" erroneous for Firefox
### What type of issue is this?

Incorrect support data (example: BrowserX says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

The "Available in service workers" item for Firefox indicate that support was added in 114

Firefox Support for regular Workers and Shared Workers were added in 114, but ES Modules are still not supported in Service Workers

### What browsers does this problem apply to, if applicable?

Firefox

### What did you expect to see?

No support

### Did you test this? If so, how?

Yes, using https://github.com/nicolo-ribaudo/import-defer-polyfill , which uses ES modules in `sw.js`

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

Bug to enable ES modules in Service Workers is https://bugzilla.mozilla.org/show_bug.cgi?id=1360870

### Do you have anything more you want to share?

-

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `javascript.statements.import`
* Report started: 2024-05-23T15:01:04.061Z

</details>

### LINKED ISSUE #23554: javascript.statements.import - not supported in ServiceWorkers in Firefox
### What type of issue is this?

Incorrect support data (example: BrowserX says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

The MDN article on the `import` statement claims that the `import` statement is "Available in service workers"_\[sic\]_ as of Firefox version 114.

### What browsers does this problem apply to, if applicable?

Firefox

### What did you expect to see?

I expected to see a service worker that used the `import` statement in its body not fail to evaluate on Firefox version 115.

### Did you test this? If so, how?

I tested this on Firefox 115.12.0esr (64-bit) on Windows.

I also tested this on Microsoft Edge version 126.0.2592.68 (Official build) (64-bit) on Windows; but the documentation isn't unambiguously wrong in that case, since Edge &mdash; [unlike Firefox](https://github.com/mdn/browser-compat-data/blob/v5.5.35/api/ServiceWorker.json#L50) &mdash; supports ["module" service workers](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/register#module), which *do* support the `import` statement.

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#browser_compatibility

To be clear, the problem is with the `import` **statement**, not the `import()` **function**, which is described on a different page and about which I have no complaints today.

### Do you have anything more you want to share?

Miminal Reproducible Example: stick these 3 files in a folder and call `python -m http.server` on it; open `localhost:8000` in Firefox, press F12, and check the console for complaints. \(Edge has the same error, but puts more informative and relevant details directly into the page's error console.\)

```html
<!-- index.html -->
<script type="module">
await navigator.serviceWorker.register('./sw.js');
</script>
```
```js
/* sw.js */
import { onmessage_sw, onfetch_sw } from "./mod.mjs";

addEventListener('message', onmessage_sw);
addEventListener('fetch', onfetch_sw);
```
```js
/* mod.mjs */
export function onmessage_sw(event) {
	console.info("Got message event: %o", event);
}

export function onfetch_sw(event) {
	console.info("Got fetch event: %o", event);
}
```

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `javascript.statements.import`
* Report started: 2024-06-28T19:39:42.992Z

</details>

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates and corrects version values for Firefox and Firefox Android for the `import.service_worker_support` member of the `statements` JavaScript feature. This fixes #23554 and fixes #23160, which contains the supporting evidence for this change.

Additional Notes: It is worth noting that this feature, and a subfeature of `ServiceWorker` indicating ECMAScript module support, were both added in #19835. It has been confirmed that ECMAScript modules aren't supported in Service Workers in Firefox yet.

### CODE REVIEW

**Comment by Josh-Cena** [comment]
Also check https://github.com/mdn/browser-compat-data/issues/23160?

File: javascript/statements.json [hunk 1]
```diff
@@ -1584,7 +1584,7 @@
               },
               "edge": "mirror",
               "firefox": {
-                "version_added": "114"
+                "version_added": false
```
Comment (Elchi3) [inline javascript/statements.json:1587]:
```suggestion
                "version_added": false,
                "impl_url": "https://bugzil.la/1360870"
```

**Review by Elchi3 — APPROVED** [review]
